api.json

```diff
@@ -0,0 +1,761 @@
+              "outreachType": "String"
+            }
+          },
+          "response": {
+            "result?": [{
+              "outreachCampaignId": "Number",
+              "organisationName": "String",
+              "userOwner": "String",
+              "outreachCampaignType": "String",
+              "outreachCampaignTarget": ["String"],
+              "createdAt": "String",
+              "modifiedAt": "String",
+              "meetings": [{
+                "meetingId": "Number",
+                "meetingType": "String",
+                "outreachPartner": "String",
+                "createdAt": "String",
+                "modifiedAt": "String"
+              }]
+            }]
+          }
+        }
+      }
+    },
+    "/users": {
+      "description": "",
+      "/": {
+        "GET": {
+          "description": "Retrieve all users accessible with the given scopes",
+          "auth": true,
+          "scope": [],
+          "query": {
+            "fields": ["String"]
+          },
+          "response": {
+            "id": "Number",
+            "name": "String",
+            "gender": "String",
+            "birthYear": "String",
+            "email": "String",
+            "phoneNumber": "String",
+            "postCode": "String",
+            "emailConsent": "Boolean",
+            "smsConsent": "Boolean",
+            "qrCodeUrl": "String",
+            "registeredAt": "String",
+            "lastActiveAt": "String",
+            "modifiedAt": "String"
+          }
+        },
+        "POST": {}
+      },
+      "/register": {
+        "POST": {
+          "auth": "Maybe - orgs/volunteers don't need auth but visitors do",
+          "scope": [],
+          "body": {
+            "organisationId": "Number",
+            "name": "String",
+            "gender": "String",
+            "birthYear": "String",
+            "email": "String",
+            "phoneNumber": "String",
+            "postCode": "String",
+            "emailConsent": "Boolean",
+            "smsConsent": "Boolean"
+          },
+          "response": {
+            "id": "Number",
+            "name": "String",
+            "gender": "String",
+            "birthYear": "String",
+            "email": "String",
+            "phoneNumber": "String",
+            "postCode": "String",
+            "emailConsent": "Boolean",
+            "smsConsent": "Boolean",
+            "qrCodeUrl": "String",
+            "registeredAt": "String"
+          }
+        }
+      },
+      "/login": {
+        "POST": {
+          "auth": null,
+          "scope": null,
+          "body": {},
+          "response": {}
+        }
+      },
+      "/:id||me": {
+        "GET": {
+          "description": "Retrieve details of a specific user",
+          "auth": true,
+          "scope": {},
+          "query": {
+            "fields": [],
+          },
+          "response": {
+            "id": "Number",
+            "name": "String",
+            "qrSecret": "String",
```

Comment:
`qrSecret`? First, why are we sending secrets in the response? Second, the QR code is only relevant for visitors, so what does this mean for others?

Reply:
☝️ same as above